website_seo_redirection/models/website_seo_redirection.py

```diff
@@ -127,3 +159,32 @@ def redirect_auto(self, path=None, code=301, website=None, rerouting=None):
             dict(request.httprequest.args),
             True,
```

Comment:
@Yajo Thats correct, I tested it with Chrome. I looked at the function redirect_with_hash(), I didn't understand the problem completely in this function but as I tested it with my temporary change it also worked in Chrome.